### PR TITLE
docs: add @ijayhub as a contributor

### PR DESCRIPTION
**Pull request by ijayhub**
## What type of PR is this? (check all applicable)

- [x] 🤝 Add a contributor

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started) section.

- [x] ✅ Yes

### How did you add yourself as a contributor?

- [x] 🤖 With CLI

#### If you added yourself manually, did you follow the [emoji key and contribution types](https://allcontributors.org/docs/en/emoji-key) to fill in the value?

- [ ] ✅ Yes
- [x] ❌ No

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?
- [x] ✅ Yes
- [ ] ❌ No

## Added to documentation?
- [ ] 🙅 no documentation needed
- [x]  yes

## Screenshot

Screenshoot OpenSauced1

## [optional] What GIF best describes this PR or how it makes you feel?
ClaireDancingGIF

### CODE REVIEW

**Comment by ijayhub** [comment]
I used CLI not manual installation

**Comment by BekahHW** [comment]
@ijayhub apologies for that. I saw you checked a box about adding manually. 

Last thing, just provide the screenshot and it should be good to go.

**Comment by adiati98** [comment]
Hey @ijayhub,

We have merged a PR before yours. Please [resolve the conflicts](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository) before we can merge yours.

And please let us know if you need any help 😊

**Comment by ijayhub** [comment]
screenshot OpenSauced

**Comment by adiati98** [comment]
> screenshot OpenSauced @BekahHW

What do you mean, @ijayhub?

**Comment by ijayhub** [comment]
😊
Screenshoot OpenSauced1 @BekahHW @adiati98

**Comment by BekahHW** [comment]
@ijayhub looking good so far. Make sure you fill out the PR template though. image

File: .all-contributorsrc [hunk 1]
```diff
@@ -1589,6 +1589,25 @@
...
+        "doc",
+        "ideas",
+        "maintenance",
+        "promotion",
+        "research",
+        "talk",
+        "tutorial",
+     ]
+   },
       "login": "Kaz-Smino",
```
Comment (adiati98) [inline .all-contributorsrc:1611]: There should be an opening bracket here.

```suggestion
   },
   {
      "login": "Kaz-Smino",
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
@ijayhub I left a review for you to fix here. :)

Then, there are a couple of things:

- After you fix it, please run the `npm run contributors:generate` as what mentioned in [the step 3 of the instruction](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=resolving-merge-conflicts). I don't see the number of contributors on the badge is changing here.
- Please answer all questions in the PR form and put your screenshot in there, too.

Thank you! Please let us know if you have any questions.

**Review by adiati98 — APPROVED** [review]
Well done, @ijayhub! 🙌
Thank you for the changes! 😄

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕 

git merge GIF

**Review by BekahHW — APPROVED** [review]

**Pull request merged**